src/index.rs

```diff
@@ -82,6 +83,8 @@ impl Entry {
 	}
 }
 
+// [Review]: #[repr(transparent)] of any use?
+/// Value address.
```

Comment:
As I understand, `repr(transparent)` concerns C ABI compatibility, which is not an issue here.